### ISSUE TITLE
Pull request angular-material so they will publish directly to Meteor

### ISSUE THREAD

**Issue by Urigo** [planet-training/meteor-angular-material#1]
Very nice work!

It will be best if you would pull request the [angular-material](https://github.com/angular/material) library to help them publish their own version to Meteor.

They use Gulp so here is a guide on how to do that:
https://github.com/MeteorPackaging/grunt-gulp-meteor

Then until they will merge your pull request you could publish from the fork with this gulp task.

Also, I would change the namespacing of the publisher to make it official, like 

```
angularjs:angular-material
```

(someone already took the 'angular' namespace...)

Send me your Meteor user name and I will add you to the angularjs Meteor organization so you can publish straight to there..

For more information:
https://github.com/MeteorPackaging
https://github.com/MeteorCommunity/discussions/issues/14
https://www.youtube.com/watch?v=g-idz8UPtDM
https://www.youtube.com/watch?v=mIxZnOtWUUY

**Comment by dohomi** [comment]
Thanks @Urigo 

I forked the angular material project, but actually they don't have the compiled sources inside of their repo. Everything is compiled inside of https://github.com/angular/bower-material but that repo I guess gets autocompiled. Will see when I get into it.

Cheers

**Comment by Urigo** [comment]
Yes it's like AngularJS..   maybe you can still pull request their gulp tasks and continue publishing here or maybe just create an issue for them with all the information

**Comment by Urigo** [comment]
@dohomi I've created the 'angular' organization and add you in so you can now publish the material library there.

**Comment by dohomi** [comment]
Hi Urigo,

I checked out the angular organization and its projects. Currently they all point to the github version of the bower project of angular. It looks that you are still maintaining the files? Would like to publish my project but I need to wait that you link urigo:angular to the angular:angular project as well, otherwise angular is getting loaded twice in my app.

How about angular:ui-calendar? I wrote a wrapper for planet training, shall I publish it there as well?

Best Regards,

Dominic

**Comment by Urigo** [comment]
Yes I get it.
I'm updating it right now and will release a new version this week.
You can update the code and wait with the release or release a new version that will only support our new version (so we won't have to wait for each other).
I also need to update other libraries that depend on angularjs:angular but let not wait and just go ahead, otherwise it might not end...

about angular:ui-calendar yes that will be great, just publish it under angularui:ui-calendar because it's an angular-ui project and not angular
